cli/src/main/scala/aqua/AquaCli.scala

```diff
@@ -47,7 +47,7 @@ object AquaCli extends IOApp with Logging {
     implicit val aio: AquaIO[F] = new AquaFilesIO[F]
     implicit val ec = r.compute
 
-    runOpt orElse versionOpt
+    createKeypairOpt orElse runOpt orElse versionOpt
```

Comment:
Maybe we can bundle the js-specific opts into `jsOpts` or smth like that, to see the clear difference of what depends on the platform and what doesn't.

Reply:
Yeah, that's what I was coming at with #376

Reply:
I think I'm not accustomed to Scala devtools enough to do this. We've discussed it with @DieMyst: he will do it in a separate PR.

Reply:
Done